tools/logstash/lib/logformat.sc

```diff
@@ -0,0 +1,34 @@
+trait LogFormat {
+  def matches(s: String): Boolean
+  val codec: String
+  val unframe: String
+}
+
+object DcosLogFormat extends LogFormat {
+  val regexPrefix = "^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}:".r
```

Comment:
Could you give an example of what is matched?

Reply:
✅